src/formatDate.js

```diff
@@ -50,7 +50,45 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const day = 'DD';
+  const month = 'MM';
+  const year = 'YYYY';
+  const shortYear = 'YY';
+
+  let oldYear = fromFormat.indexOf(year);
+  const oldMonth = fromFormat.indexOf(month);
+  const oldDay = fromFormat.indexOf(day);
```

Comment:
Fix naming everywhere
```suggestion
  let oldYearIndex = fromFormat.indexOf(year);
  const oldMonthIndex = fromFormat.indexOf(month);
  const oldDayIndex = fromFormat.indexOf(day);
```